### DISCUSSION TITLE
posts/why-i-choose-funtional-options-pattern-instead-of-builder-pattern-for-golang/

### DISCUSSION THREAD

**Discussion by giscus** [qtsky89/qtsky89.github.io#9] [General]
# posts/why-i-choose-funtional-options-pattern-instead-of-builder-pattern-for-golang/

Summary

https://techblog.wonhee.net/posts/why-i-choose-funtional-options-pattern-instead-of-builder-pattern-for-golang/

**Comment by seulgioh** [comment]
Awesome :) 👍
